pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1966,6 +1991,7 @@ public void deleteRealtimeTable(String tableName, @Nullable String retentionPeri
         InstancePartitionsType.CONSUMING.getInstancePartitionsName(rawTableName));
     InstancePartitionsUtils.removeInstancePartitions(_propertyStore,
         InstancePartitionsType.COMPLETED.getInstancePartitionsName(rawTableName));
+    InstancePartitionsUtils.removeTierInstancePartitions(_propertyStore, rawTableName);
```

Comment:
nit: just to be consistent with the method above, move this one below the info log and add info log for deleting tier instance partitions?

Reply:
Ack